core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1076,6 +1076,32 @@ public static int levenshtein(String string1, String string2) {
     return LEVENSHTEIN_DISTANCE.apply(string1, string2);
   }
 
+  /** SQL FIND_IN_SET(string, stringArray) function. */
+  public static int findInSet(String matchStr, String textStr) {
+    if (matchStr.contains(",")) {
```

Comment:
Why you add this check? Could you give the code comments to explain?

Reply:
This check is from the function specification, which refered to [Spark's doc](https://spark.apache.org/docs/latest/api/sql/index.html#find_in_set). I will add function annotation to address the meaning of this function.